scipy/stats/_multivariate.py

```diff
@@ -2004,14 +2019,15 @@ def _standard_rvs(self, n, shape, dim, df, random_state):
             size=n*n_tril).reshape(shape+(n_tril,))
 
         # Random chi-square variates for diagonal elements
-        variances = np.r_[[random_state.chisquare(df-(i+1)+1, size=n)**0.5
-             for i in range(dim)]].reshape((dim,) + shape[::-1]).T
+        variances = (np.r_[[random_state.chisquare(df-(i+1)+1, size=n)**0.5
```

Comment:
Some parts of this module put spaces around operators like `+`, but not i.e., here. I think PEP8 doesn't actually have a hard rule on that; maybe better not to go down that rabbit hole in any case.

Reply:
I'm not against more spaces.  I think I just started running out of steam, and didn't want to deal with a line that would probably then exceed 80 characters.  And I think I'll still leave this for the next clean up.